ground/gcs/src/plugins/rawhid/rawhidplugin.h

```diff
@@ -95,7 +92,7 @@ class RAWHID_EXPORT RawHIDPlugin
     virtual bool initialize(const QStringList &arguments, QString *error_message);
     virtual void extensionsInitialized();
 private:
-    RawHIDConnection *hidConnection;
+    RawHIDConnection* hidConnection;
```

Comment:
It's supposedly qt style to group with the type name and is consistent with
the other declarations nearby I think.

I hate it too.

On Mon, Mar 21, 2016, 12:04 PM Michael Corcoran <notifications@github.com>
wrote:

> In ground/gcs/src/plugins/rawhid/rawhidplugin.h
> <https://github.com/d-ronin/dRonin/pull/801#discussion_r56878221>:
>
> > @@ -95,7 +92,7 @@ class RAWHID_EXPORT RawHIDPlugin
> >      virtual bool initialize(const QStringList &arguments, QString *error_message);
> >      virtual void extensionsInitialized();
> >  private:
> > -    RawHIDConnection *hidConnection;
> > +    RawHIDConnection* hidConnection;
>
> Y you do dis?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/d-ronin/dRonin/pull/801/files/e0dc6bbbe03abea8536522066c3a379988660d2a..a179530519cea192514f036b123a7a25e370b8ee#r56878221>
>